src/library/scala/sys/BooleanProp.scala

```diff
@@ -63,12 +63,13 @@ object BooleanProp {
   def valueIsTrue[T](key: String): BooleanProp = new BooleanPropImpl(key, _.toLowerCase == "true")
 
   /** As an alternative, this method creates a BooleanProp which is true
-   *  if the key exists in the map.  This way -Dfoo.bar is enough to be
-   *  considered true.
+   *  if the key exists in the map and is not assigned a value other than "true",
```

Comment:
You have to read the comment in a mirror. It's written from the standpoint of a falsifier, kind of like what do you ask a Cretan liar. How do I turn it off? Also, everyone knows that Father's Day is also Opposite Day.